src/main/java/org/cru/godtools/domain/packages/PageStructure.java

```diff
@@ -284,6 +306,45 @@ private void removeXmlContent(Node baseContentElement, Node updatedContentElemen
 		}
 
 		return;
+		
+	}
+
+	public void updateXmlContentAttributes(Document updateXmlDocument) throws TransformerException, IOException
```

Comment:
@LaelWatkins It would be nice if this can be done recursively like the other two methods, but if this works we can move ahead w/ it for now.